$NetBSD: patch-src_polkit-mate-authentication-agent-1.desktop.in.in,v 1.1 2018/06/14 21:29:36 youri Exp $

Run the MATE polkit agent in other desktops that provide one.

--- src/polkit-mate-authentication-agent-1.desktop.in.in.orig	2018-01-28 07:54:19.000000000 +0000
+++ src/polkit-mate-authentication-agent-1.desktop.in.in
@@ -6,5 +6,5 @@ Terminal=false
 Type=Application
 Categories=
 NoDisplay=true
-OnlyShowIn=MATE;
+OnlyShowIn=MATE;X-Cinnamon;XFCE;LXDE;LXQT;
 X-MATE-AutoRestart=true
